llmfoundry/data/finetuning/dataloader.py

```diff
@@ -145,6 +148,14 @@ def build_finetuning_dataloader(cfg: DictConfig, tokenizer: Tokenizer,
         )
 
     else:
+        if '://' in cfg.dataset.hf_name:
```

Comment:
Worth getting a check from @eracah on this pattern. I think Composer already has utilities for checking whether a path points to a remote object store supported by `get_file`.